news/11314.bugfix.rst

```diff
@@ -0,0 +1 @@
+Avoid AttributeError when _distutils_hack is missing its implementation.
```

Comment:
```suggestion
Avoid ``AttributeError`` when removing the setuptools-provided ``_distutils_hack`` is missing its implementation.
```

Reply:
Should that be:

Avoid ``AttributeError`` when removing the setuptools-provided ``_distutils_hack`` and it is missing its implementation.

Reply:
@bdrosen96 can you please update the news fragment as you suggest and squash the commits in this PR? This will the go in 22.2.2.

Reply:
Done